packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -15,6 +15,7 @@
   box-sizing: border-box;
   display: flex;
   position: relative;
+  border: 1px solid transparent;
```

Comment:
Yeah, but we aren't setting a width/height on these, they are just pushed out by their contents. 

It's not a huge deal, but users will see their message bars shift by 1 pixel. Look at these 2 message bars, one with an exaggerated border

http://codepen.io/micahgodbolt/pen/YZbKEV

![image](https://cloud.githubusercontent.com/assets/1434956/24730862/19937876-1a1b-11e7-806f-a52e729a6804.png)

Fortunately transparent borders won't leave blank white space, but they will push the content in 1 pixel, and increase the overall height of the bar by 2 pixels.